src/HMR/client/utils.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ */
+
+/* @flow */
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-unused-expressions */
+
+// $FlowFixMe
+import { Platform } from 'react-native';
+
+export default function resetRedBox() {
+  if (Platform.OS === 'ios') {
+    const RCTRedBox = require('NativeModules').RedBox;
+    RCTRedBox && RCTRedBox.dismiss && RCTRedBox.dismiss();
```

Comment:
Is it possible that `RCTRedBox` and `RCTExceptionsManager` may not exist?

nit: let's remove the `RCT` prefix. It's a convention on Objective-C side. looks weird in JS